src/ctia/schemas/graphql/helpers.clj

```diff
@@ -88,10 +111,12 @@
                                    values
                                    default-type-registry))
   ([^String enum-name ^String description values registry]
-   (or (get @registry enum-name)
-       (let [builder (-> (GraphQLEnumType/newEnum)
+    (get-or-update-type-registry
+      registry
+      enum-name
+      #(let [builder (-> (GraphQLEnumType/newEnum)
                          (.name enum-name)
-                         (.description description))
+                         (.description ^String description))
              names-and-values? (map? values)]
```

Comment:
Please temporarily permit the funny indentation, a forthcoming commit will wrap this entire return value in a function and restore some order.